### ISSUE TITLE
Vimeo: Downloaded video is missing a section

### ISSUE THREAD

**Issue by rsnitsch** [ytdl-org/youtube-dl#29035]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2021.04.26**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
λ youtube-dl.exe -v -f mp4 https://player.vimeo.com/video/14907669
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', '-f', 'mp4', 'https://player.vimeo.com/video/14907669']
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, pref cp1252
[debug] youtube-dl version 2021.04.26
[debug] Python version 3.9.1 (CPython) - Windows-10-10.0.19041-SP0
[debug] exe versions: ffmpeg 4.2.2, ffprobe 4.2.2
[debug] Proxy map: {}
[vimeo] 14907669: Downloading webpage
[vimeo] 14907669: Downloading akfire_interconnect_quic m3u8 information
[vimeo] 14907669: Downloading akfire_interconnect_quic m3u8 information
[vimeo] 14907669: Downloading fastly_skyfire m3u8 information
[vimeo] 14907669: Downloading fastly_skyfire m3u8 information
[vimeo] 14907669: Downloading akfire_interconnect_quic MPD information
[vimeo] 14907669: Downloading akfire_interconnect_quic MPD information
[vimeo] 14907669: Downloading fastly_skyfire MPD information
[vimeo] 14907669: Downloading fastly_skyfire MPD information
[debug] Invoking downloader on 'https://skyfire.vimeocdn.com/1620920286-0x6ca57f9108212dec36ec787e56a5c7d2d19a3bbd/14907669/video/61014879,28342027/master.mpd?base64_init=1'
[dashsegments] Total fragments: 225
[download] Destination: The Jewish Community of Sweden-14907669.mp4
[download] 100% of 119.66MiB in 00:48
```

## Description

The downloaded video is missing a section that starts at 20:13 when viewed in the browser (the clip with Barbara Lerner Spectre):
https://player.vimeo.com/video/14907669. I am using Microsoft Edge browser.

Note that when you play the video in the browser with the alternative URL https://vimeo.com/14907669 - then the clip is also missing in the browser. I'm curious what's the issue here.

So far I've only been able to save the complete video by screen recording it.

**Comment by kikuyan** [comment]
If you don't mind the quality, you'll get longer video by downloading 400x320 version.
640x480 - 225 segments 23:38 duration
400x320 - 235 segments 24:51 duration

Only downloaded DASH and don't know HLS.

**Comment by rsnitsch** [comment]
> If you don't mind the quality, you'll get longer video by downloading 400x320 version.
> 640x480 - 225 segments 23:38 duration
> 400x320 - 235 segments 24:51 duration
> 
> Only downloaded DASH and don't know HLS.

Thank you for the hint! I downloaded all formats and checked their length. This is the result:
![image](https://user-images.githubusercontent.com/1746428/118400701-ffd2fe00-b662-11eb-9d12-dd653d028018.png)

It's really weird that the section that I'm interested in is missing in about half of the formats... it's included in the lower quality formats, but not in the higher quality ones. I wonder why...

Anyway, my issue is now solved and I guess it's Vimeo's fault and not youtube-dl's. Therefore I am closing this issue. Thanks again @kikuyan.
